### ISSUE TITLE
Webpage visor and actions to control navigation

### ISSUE THREAD

**Issue by arasaac-dga** [asterics/AsTeRICS-Grid#296]
Add a webpage visor and actions like Youtube that allow to navigate webpages without leave the communicator.

**Comment by klues** [comment]
With "webpage visor" you mean an element like the YouTube-Element where any custom website can be shown and the user can navigate using buttons like "next link/previous link/click", similar to the "play/pause/stop" elements for the YouTube player. Correct?!

**Comment by arasaac-dga** [comment]
Yes, exactly. That's the idea. And buttons to scroll up/down and to open a website directly too.

**Comment by klues** [comment]
It was my plan to implement this within my netidee funding, however it turned out that it's not really possible from a technical point of view. Most webpages prevent being embedded in an iframe nowadays, so this would only work for a very limited set of webpages (and not the big, important ones). Therefore it's not really possible or makes not much sense to implement it.

**Comment by arasaac-dga** [comment]
Ok, we understand. Doesn't matter. Being a technical problem with webpages, there is little that can be done. Thanks anyway to consider.
